test/requirements-test.txt

```diff
@@ -16,8 +16,7 @@ git+https://github.com/microsoft/TransformerCompression.git ; python_version >=
 mlflow>=2.4.0
 neural-compressor
 nncf==2.7.0
-nvidia-modelopt~=0.11.0
-onnx-graphsurgeon
+nvidia-modelopt[onnx]~=0.11.0
```

Comment:
can you change this to the previous `nvidia-modelopt~=0.11.0` and `onnx-graphsurgeon`. 
Like we discussed earlier, the `onnx` extra in the ci test would install the ort dependencies from this extra on top of what ort is already installed in the test environment.